src/djangooidc/tests/test_views.py

```diff
@@ -228,6 +238,128 @@ def test_login_callback_initializes_client_and_succeeds(self, mock_client):
                         # assert that redirect is to / when no 'next' is set
                         self.assertEqual(response.status_code, 302)
                         self.assertEqual(response.url, "/")
+    
+    @less_console_noise_decorator
+    def test_login_callback_sets_verification_type_regular(self, mock_client):
```

Comment:
**(Very smol nitpick):** Would it be possible to leave a comment here mentioning that REGULAR meaning "Verified by Login.gov"?